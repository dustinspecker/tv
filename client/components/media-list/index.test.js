import MediaList from '.'
import React from 'react'
import renderer from 'react-test-renderer'
import {StaticRouter} from 'react-router'

test('MediaList displays list of shows', () => {
  const props = {
    media: [{name: 'show1', show: '/show1'}, {name: 'show2', show: '/show2'}]
  }

  const component = renderer.create(
    <StaticRouter context={{}}>
      <MediaList {...props} />
    </StaticRouter>
  )

  expect(component.toJSON()).toMatchSnapshot()
})
