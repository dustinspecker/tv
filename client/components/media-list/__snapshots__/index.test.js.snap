// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MediaList displays list of shows 1`] = `
<ul
  className="MuiList-root-1 MuiList-padding-2"
>
  <a
    href="/show/show1"
    onClick={[Function]}
  >
    <li
      className="MuiListItem-root-5 MuiListItem-default-8 MuiListItem-gutters-13"
      disabled={false}
    >
      <div
        className="MuiListItemText-root-17"
      >
        <span
          className="MuiTypography-root-23 MuiTypography-subheading-30 MuiListItemText-primary-20"
        >
          show1
        </span>
      </div>
    </li>
  </a>
  <a
    href="/show/show2"
    onClick={[Function]}
  >
    <li
      className="MuiListItem-root-5 MuiListItem-default-8 MuiListItem-gutters-13"
      disabled={false}
    >
      <div
        className="MuiListItemText-root-17"
      >
        <span
          className="MuiTypography-root-23 MuiTypography-subheading-30 MuiListItemText-primary-20"
        >
          show2
        </span>
      </div>
    </li>
  </a>
</ul>
`;
